
geometry_msgs/Quaternion rotation
MoveOptions options
